RELEASE_NOTES.md

```diff
@@ -105,22 +105,22 @@ making a backup is strongly advised.
   and a new color picker which creates a curve based on selected image area.
 
 - A new module 'filmic RGB' which, like the previous 'filmic', is designed
-  to replace the 'base curve', 'shadows and highlights' modules and other tone-mapping options. 
+  to replace 'base curve', 'shadows and highlights' and other tone-mapping modules. 
   This new version replaces the one introduced in 2.6.2; it should be easier to use,
   and it will reduce color casts. The old 'filmic' module is now deprecated and
   is only available on images where it was already used for editing.
 
-- A new module 'tone equalizer', designed to merge the features of 'local tone mapping', 
+- A new module 'tone equalizer', designed to merge the features of 'tone mapping', 
```

Comment:
the module is called "tone mapping", but it's actually a local tone mapping, so it should be put at least : 
(local) 'tone mapping'

Reply:
Ok. About that distinction: should filmic rgb be said to replace other *global* tone mapping modules, or is that too specific in that case?

Reply:
Yes, indeed, filmic is a global tone mapping.

Reply:
Done.